src/biogeochem/VOCEmissionMod.F90

```diff
@@ -85,14 +85,14 @@ module VOCEmissionMod
   !------------------------------------------------------------------------
   subroutine Init(this, bounds)
 
-    use clm_varctl, only : use_fates, use_fates_nocomp
+    use clm_varctl, only : use_fates, use_fates_nocomp, use_fates_sp
     class(vocemis_type) :: this
     type(bounds_type), intent(in)    :: bounds  
 
 
     if ( shr_megan_mechcomps_n > 0) then
        if (use_fates) then
-          if (.not. use_fates_nocomp) then
+          if (.not. (use_fates_nocomp .or. use_fates_sp)) then
```

Comment:
I don't think we need this (NOCOMP is always on when SP is on)

Reply:
Was reverted.